src/components/UI/atoms/Button/Button.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import widths from '<variables>/width';
+import themeBorderRadius from '<variables>/border';
+import heights from '<variables>/height';
+import { buttonColors } from '<variables>/colorPalette';
+import { fontSizes } from '<variables>/fonts';
+
+/**
+ * @description - Button Component
+ *
+ * @prop {string} display - display
+ * @prop {string} hoverBackground - hoverBackground
+ * @prop {string} hoverColor - hoverColor
+ * @prop {string} fontSize - font Size
+ * @prop {string} borderColor - border
+ * @prop {string} borderRadius - border radius
+ * @prop {string} width - width
+ * @prop {string} height - height
+ * @prop {string} color - color
+ * @prop {function} onClick - onClick
+ * @prop {string} backgroundColor - background color
+ * @prop {children} children - children
+ *
+ * @return {component} Button
+ */
+const Button = ({
+  hoverBackground,
```

Comment:
Is there any occasions in the designs the hover will change. I don't know why you are passing them in as props

Reply:
Yes, the get started button has a different hover color, from the other buttons, because of the color scheme used for that button.

Reply:
Can you not pass a button type and detect these properties. So if the button type is a start button it has a different color. Imagine if you have a danger button, would you be passing those props as well, or a secondary button. So check for the button type passed and render its additional properties.

Reply:
Oh, a buttonType prop would actually be a better way to go about it. 
It would abstract the number of props needed.
I would implement that.
Thank you.